netci.groovy

```diff
@@ -374,7 +430,7 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                         }
                     }
                     // For x86, only add per-commit jobs for Windows
-                    else if (architecture == 'x86ryujit' || architecture == 'x86lb') {
+                    else if (architecture == 'x86' || architecture == 'x86compatjit' || architecture == 'x86lb') {
```

Comment:
Do we need to run x86 compatjit as a PR trigger? Seems like this and legacy backend can now be outerloop tests run either per checkin or on a timer.